src/index.css

```diff
@@ -1,3 +1,27 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;600;700&family=Roboto+Slab:wght@300;400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  color: inherit;
+}
+
 body {
+  padding: 4%;
+  background-color: #f5f6fa;
+}
+
+#root {
   margin: 0;
+  padding: 0 0 8% 0;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #fafafa;
+}
+
+.container {
+  padding: 4%;
 }
```

Comment:
- [ ] The style is as the Design but you should remove the spaces in the edges cause it doesn't make sense for a website like that cause the nav bar should be at the top so please try to remove the spaces indicated in the picture below so that you match the Design 
<img width="1440" alt="Screenshot 2023-04-07 at 17 15 18" src="https://user-images.githubusercontent.com/61790587/230650345-5f0c131d-c961-4bef-8335-0f364f7b33dd.png">